EnrollmentFlows/SignUp/footer.html

```diff
@@ -0,0 +1,206 @@
+<div id="cookies">
+    <p>
+        <a id="js-accept-cookies" class="cookies-ok" href="#">OK </a>
+        We use cookies to identify you, thereby enabling us to grant you access to the services and resources provided by the Infrastructure, and also to improve your experience on our site. To find out more, read our
+        <a href="policy" target="_blank">updated privacy policy and cookie policy.</a>
+    </p>
+</div>
+
+<footer class="ssp-footer text-center">
+    <div class="mdl-grid  ssp-footer__item">
+        <div class = "mdl-cell mdl-cell--4-col mdl-cell--12-col-phone mdl-cell--12-col-tablet">&nbsp;</div>
+        <div class = "mdl-cell mdl-cell--4-col mdl-cell--12-col-phone mdl-cell--12-col-tablet mdl-cell__images">
+            <a href="https://www.grnet.gr">
+                <img class="ssp-footer__item__logo" src="https://rciam.aai-dev.grnet.gr/ui2/module.php/themeegi/resources/images/grnet_logo_en.svg" alt="GRNET">
+            </a>
+            <img class="ssp-footer__item__logo--eu" src="https://rciam.aai-dev.grnet.gr/ui2/module.php/themeegi/resources/images/eu.svg" alt="European Union">
```

Comment:
I would turn this into a link:
`footer.html --> ../footer.html`
unless the Enrollment UI theme footer is different

Reply:
Done